### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by berenss**
minor edits.
brew didn't find 'wait' on my mac, 
% brew install wait
Updating Homebrew...
Error: No available formula with the name "wait" 
==> Searching for a previously deleted formula (in the last month)...
Warning: homebrew/core is shallow clone. To get complete history run:
  git -C "$(brew --repo homebrew/core)" fetch --unshallow

Error: No previously deleted formula found.
==> Searching for similarly named formulae...
These similarly named formulae were found:
adwaita-icon-theme                   juju-wait                            kqwait                               wait_on
To install one of them, run (for example):
  brew install adwaita-icon-theme
==> Searching taps...
==> Searching taps on GitHub...
Error: No formulae found in taps.

Also before we run the start.sh, I believe it is necessary to have an export KUBECONFIG and oc login for the cluster.

**Description of the change:**

**Motivation for the change:**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ You can find information on how to contribute to this project and our docs proje
 - `oc` is connected with adequate permissions to create new namespaces in your OCP cluster.
 - macOS users:
    - `gsed` is required. Install using `brew install gnu-sed`
-   - `wait` is optional. Install using `brew install wait`
+   - `wait` is optional. Install using `brew install juju-wait`
```
Comment (tpouyer) [inline README.md:33]: I think this should be `brew install watch`

File: README.md [hunk 2]
```diff
@@ -79,6 +79,8 @@ Either way you choose to go you're gonna need a `pull-secret`... we are still in
 
 We've added a very simple `start.sh` script to make your life easier... if you want to deploy `OCM` the __"hard way"__ you can find the instructions for deploying `OCM` using nothing but `oc` commands [here](#manually-deploy-using-oc-commands-the-hard-way).
 
+0. First need to export KUBECONFIG=/path/to/some/cluster/kubeconfig
+
```
Comment (tpouyer) [inline README.md:83]: The README already points out that `oc` and `kubectl` must be configured to connect to your target OCP cluster:  https://github.com/open-cluster-management/deploy#prereqs

File: README.md (hunk 1 above)
Comment (berenss) [reply README.md:33]: got it, that makes sense.

File: README.md (hunk 2 above)
Comment (berenss) [reply README.md:83]: "oc & kubectl (ver. 1.16+) configured to connect to your OCP cluster"

we could make this a bit easier by showing the `export KUBECONFIG` and an `oc login`
either way, the script will fail right away if it doesn't make a cluster connection.

**Pull request merged**